src/plugins/importedRootPlugin/ImportModelProvider.js

```diff
@@ -0,0 +1,78 @@
+define([
+    '../../api/objects/object-utils'
+], function (
+    objectUtils
+) {
+    /**
+     * Transforms an import json blob into a object map that can be used to
+     * provide objects.  Rewrites root identifier in import data with provided
+     * rootIdentifier, and rewrites all child object identifiers so that they
+     * exist in the same namespace as the rootIdentifier.
+     */
+    function rewriteObjectIdentifiers(importData, rootIdentifier) {
```

Comment:
[The Import action](https://github.com/nasa/openmct/blob/bfa3bbcdc78cfdba62b54c0119961a7138e80301/platform/import-export/src/actions/ImportAsJSONAction.js) needs to go through the same identifier-rewriting process. Maybe pull that out into a service, in the spirit of DRY?

Reply:
The reason I wouldn't do this is because I'm not certain this is standardized (and it's already immediately broken by new plot).

HRRRRM... sounds like a new ticket.